src/main/java/echo/Server.java

```diff
@@ -0,0 +1,91 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+
+
+public class Server {
+    public static void main(String[] args) throws IOException {
+        runEchoServer();
+    }
+
+    public static ServerSocket serverSocket = null;
+        public static Socket clientSocket = null;
+
+
+
+    public static void runEchoServer() throws IOException {
+
+        try {
+            serverSocket = createServerSocket(7777);
+
+        } catch (IOException e) {
+            errorMessage("Can't listen on port number 7777", e);
+
+        }
+
+        print("Listening for connection...");
+
+        try {
+
+            clientSocket = connectClient(serverSocket);
+
+        } catch (IOException e) {
+            errorMessage("***Accept failed!!!***", e);
+
+
+        }
+
+        print("Connection Successful");
+        print("Listening for input");
+
+        echoMessage(clientSocket, serverSocket);
+
+
+    }
+
+    public static ServerSocket createServerSocket(int port) throws IOException {
+        ServerSocket serverSocket = new ServerSocket(port);
+        print("***Successful*** " + serverSocket);
+        return serverSocket;
+    }
+
+    public static Socket connectClient(ServerSocket serverSocket) throws IOException {
+        Socket clientSocket = serverSocket.accept();
+        return clientSocket;
+    }
+
+    public static void errorMessage(String message, IOException e) {
```

Comment:
this was surprising to me - the `errorMessage` not only print it, but also exits the program. I think these two functionalities should be separate.